kong/plugins/httplog/log.lua

```diff
@@ -0,0 +1,19 @@
+local cjson = require "cjson"
+local http_client = require "kong.tools.http_client"
+
+local APPLICATION_JSON_TYPE = "application/json"
```

Comment:
agreed, in the future if there would be format transformers, at that point it would be appropriate to add this back.